starter-code/main.js

```diff
@@ -0,0 +1,292 @@
+              founded_month: 2
+            }, {
+              name: 1,
+              founded_year: 1,
+              founded_month: 1
+            }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              }
+            });
+            break;
+          case "5":
+            db.collection('companies').find({
+              founded_year: 2004,
+              $and: [{
+                  "founded_month": {
+                    $gte: 4
+                  }
+                },
+                {
+                  "founded_month": {
+                    $lte: 6
+                  }
+                }
+              ]
+            }, {
+              name: 1,
+              founded_year: 1,
+              founded_month: 3
+            }).sort({
+              name: 1
+            }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              }
+            });
+            break;
+          case "6":
+            db.collection('companies').find({
+              "offices.city": "Barcelona"
+            }, {
+              name: 1
+            }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              }
+            });
+            break;
+          case "7":
+            db.collection('companies').find({}, {
+              number_of_employees: {
+              	//??? not sure about this
+                $slice: 10
+              }
+            }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              }
+            });
+            break;
+          case "8":
+            db.collection('companies').find({
```

Comment:
👍 you didn't need the name 1 
**db.collection('companies').find({name: "Facebook"}).toArray((error, result)=>{**